### PR TITLE
chore: sync fork with upstream

### PR DESCRIPTION
**Pull request by rgimen3z**
`git fetch upstream`

`git merge upstream`

### CODE REVIEW

File: demo/tests.py [hunk 1]
```diff
@@ -74,4 +74,19 @@ def test_menu_links(client: TestClient, url: str):
     assert isinstance(data, list)
 
 
+# def test_forms_validate_correct_select_multiple(client: TestClient):
```
Comment (jstebbssharp) [inline demo/tests.py:77]: Is this meant to be commented out?

File: src/npm-fastui/src/components/display.tsx [hunk 2]
```diff
@@ -26,6 +29,28 @@ export const DisplayComp: FC<Display> = (props) => {
   }
 }
 
+// todo: this list should probably be defined in the models file
```
Comment (jstebbssharp) [inline src/npm-fastui/src/components/display.tsx:32]: nit - todo still appliable?

File: demo/tests.py (hunk 1 above)
Comment (rgimen3z) [reply demo/tests.py:77]: not sure, this is coming from the upstream (https://github.com/pydantic/FastUI/tree/main)

File: src/npm-fastui/src/components/display.tsx (hunk 2 above)
Comment (rgimen3z) [reply src/npm-fastui/src/components/display.tsx:32]: same here (https://github.com/pydantic/FastUI/tree/main)

**Review by jstebbssharp — APPROVED** [review]
lgtm

**Review by jstebbssharp — APPROVED** [review]
double?

**Pull request merged**